### PR TITLE
chore(architecture): removed template code

### PR DESCRIPTION
**Pull request by pilg5971**
Removing the svelte app demo code and adding a header element for page navigation via bootstrap:

image

### CODE REVIEW

File: CribbageTournamentTracker/package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
 		"typescript": "^5.0.0",
 		"vite": "^4.4.2"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"bootstrap": "^5.3.1"
+	}
```
Comment (pilg5971) [inline CribbageTournamentTracker/package.json:35]: Adding bootstrap dependency

File: CribbageTournamentTracker/src/app.html [hunk 2]
```diff
@@ -3,10 +3,12 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
+		<link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css" />
 		<meta name="viewport" content="width=device-width" />
 		%sveltekit.head%
 	</head>
 	<body data-sveltekit-preload-data="hover">
 		<div style="display: contents">%sveltekit.body%</div>
+		<script src="../node_modules/bootstrap/dist/js/bootstrap.bundle.js" async defer></script>
```
Comment (pilg5971) [inline CribbageTournamentTracker/src/app.html:12]: mapping the dependency to our code base

File: CribbageTournamentTracker/src/routes/Header.svelte [hunk 3]
```diff
@@ -1,129 +1,44 @@
...
+		<div class="collapse navbar-collapse" id="navbarNav">
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					<a class="nav-link active" aria-current="page" href="/">Home</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="/about">About</a>
+				</li>
+			</ul>
+		</div>
```
Comment (pilg5971) [inline CribbageTournamentTracker/src/routes/Header.svelte:39]: Bootstrap navbar template obtained here: https://getbootstrap.com/docs/5.0/components/navbar/

**Review by BrandonDedolph — APPROVED** [review]
Very Nice

**Pull request merged**